app/views/vehicles/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Create a New Vehicle</h1>
+<form action="/vehicles" method="post">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
```

Comment:
I'm interested to know what this hidden input field is doing with the authenticity token. Can you help me understand what is going on here better?

Reply:
My basic understanding is that post and patch requests taking params from a form require authenticity tokens by default as a basic security feature.  I'm not sure how exactly to bypass that nor if it would be a good idea.

Reply:
Ok this confusion is because I am using the [form for helper](https://github.com/rails/rails/blob/master/guides/source/form_helpers.md). This token is usually generated by the helper. When I put my PR up I will show you.

Reply:
You are right I think about the authenticity token.